src/__tests__/index.js

```diff
@@ -299,8 +299,43 @@ const shortTests = [
   [[{mozTransformOrigin: '10% 50%'}], {mozTransformOrigin: '90% 50%'}],
   [[{MozTransformOrigin: '10% 50%'}], {MozTransformOrigin: '90% 50%'}],
   [
-    [{animationName: [{from: {transform: 'rotate(100deg)'}, to: {transform: 'rotate(-100deg)'}}]}],
-    {animationName: [{from: {transform: 'rotate(-100deg)'}, to: {transform: 'rotate(100deg)'}}]}
+    [
+      {
+        animationName: [
+          {
+            from: {transform: 'rotate(100deg)'},
+            to: {transform: 'rotate(-100deg)'},
+          },
+        ],
+      },
+    ],
+    {
+      animationName: [
+        {
+          from: {transform: 'rotate(-100deg)'},
+          to: {transform: 'rotate(100deg)'},
+        },
+      ],
+    },
+  ],
+  [[{transition: 'margin-right 4s'}], {transition: 'margin-left 4s'}],
+  [
+    [{transition: 'padding-left 4s ease-in-out'}],
+    {transition: 'padding-right 4s ease-in-out'},
+  ],
+  [[{transition: 'all 0.5s ease-out'}], {transition: 'all 0.5s ease-out'}],
+  [[{transition: 'inherit'}], {transition: 'inherit'}],
+  [[{transition: 'initial'}], {transition: 'initial'}],
+  [[{transition: 'unset'}], {transition: 'unset'}],
```

Comment:
Several of these belong in the "unchanged" array below. Could you move them please?

Reply:
Done!